com.bitwig.BitwigStudio.yaml

```
@@ -0,0 +1,76 @@
+---
+app-id: com.bitwig.BitwigStudio
+command: bitwig-studio
+runtime-version: "20.08"
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+rename-desktop-file: bitwig-studio.desktop
+rename-icon: bitwig-studio
+#rename-appdata-file: 'bitwig-studio.appdata.xml'
+finish-args:
+  # We need MIDI, Audio, USB, DRI, and realtime performances
+  - --device=all
+
+  # Needed to talk with X11, Wayland, pulseaudio and pipewire
+  - --share=ipc
+  - --filesystem=xdg-run/pipewire-0
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=x11
+
+  # Needed for updating the sound content, checking for online notifications and authentication
+  - --share=network
+
+  # For desktop notifications
+  - --talk-name=org.freedesktop.Notifications
+
+  # Give access to HOME
+  - --filesystem=home
+
+  # Java stuff
+  - --persist=~/.java
+  - --persist=~/.BitwigStudio
+  - --persist=~/Bitwig\ Studio
```

As I suspected, flatpak treats `\` here literally, so the backslash should be removed. Also `--presist` shouldn't start with `~/` (`--persist=` arg always refers to a directory under `~/`), sorry for not noticing this right away. I believe this should be changed like this:
```suggestion
  - --persist=.java
  - --persist=.BitwigStudio
  - --persist=Bitwig Studio
```

Also since the last one is a user-visible directory, maybe it should be `--filesystem=~/Bitwig Studio:create`, so it would appear in the real user home directory?